apps/site/assets/css/_location-card.scss

```diff
@@ -21,65 +18,54 @@
 .c-location-card {
   background-color: $white;
   border: 1px solid $gray-lightest;
-  border-radius: $space-1 * 3;
-  color: $gray-dark;
-  display: inline-block;
-  margin: $base-spacing / 2;
   padding: $base-spacing;
-  width: 100%;
-}
 
-.c-location-card--flex {
-  @include media-breakpoint-only(xs) {
-    flex: 1 1 100%;
-    margin: .25rem;
-    padding: .5rem 1rem;
+  &__place {
+    font-size: 1.5rem;
+    font-weight: $headings-font-weight;
+    line-height: 1.9rem;
+    @include media-breakpoint-down(xs) {
+      font-size: 1.15rem;
+    }
+    width: 67%;
+    overflow: hidden;
   }
 
-  @include media-breakpoint-only(sm) {
-    flex: 1 1 33%;
-    margin: .25rem;
-    padding: .5rem 1rem;
+  &__distance {
+    font-size: $font-size-base-xxl;
+    text-align: right;
+    @include media-breakpoint-down(xs) {
+      font-size: $font-size-base-sm;
+    }
+    width: 33%;
   }
 
-  @include media-breakpoint-only(md) {
-    flex: 1 1 33%;
-    padding: .5rem 1rem;
+  &__location {
+    font-size: $font-size-base-xxl;
```

Comment:
For the font-size and line-height: I'm looking at the mockup and it seems like these are just the standard body font size and line height... do they really need customization here?

Reply:
Hmm, when I toggle these attributes in the dev tools, the style visibly changes, so it seems that the standard sizes aren't being applied.  I'm not sure what's not connecting, then...  What would you do?

Reply:
Oh, I do see the difference too! It just matches the mockup better to me without these additions, but maybe it looks better for you with these additions... 😅 maybe it's for Yinka to decide anyway.

Reply:
Yinka had me make a few tweaks  👍